freeipa.spec.in

```diff
@@ -961,6 +967,8 @@ find \
 	-type f -exec grep -qsm1 '^#!.*\bpython' {} \; \
 	-exec sed -i -e '1 s|^#!.*\bpython[^ ]*|#!%{__python2}|' {} \;
 
+aclocal --force
```

Comment:
Why is this necessary? The tar balls should contain configure script.

Reply:
The build issues that we had on F-29 are because of not having aclocal v1.15. There is only aclocal v1.16. If you are adding patches, then it is needed to run aclocal and autoconf with the force option.

Reply:
@tiran we've been hit twice already in F28 lifetime when autotools were upgraded in Fedora, first to 1.15 and now to 1.16. Actual autotools version is hardcoded in the generated script so you need to have exact environment to the one where tarball was generated or you have to rebuild it.

Reply:
We also have to re-run aclocal/autoconf for any change with Makefile.am, so let us just re-run it always.

Reply:
Why not use autoreconf -ivf to ensure everything is up to date?